core/src/main/java/hudson/model/View.java

```diff
@@ -114,7 +117,9 @@
 import java.util.logging.Logger;
 
 import static javax.servlet.http.HttpServletResponse.SC_BAD_REQUEST;
-import static jenkins.model.Jenkins.*;
+import static jenkins.model.Jenkins.checkGoodName;
```

Comment:
For whatever reason, the wildcard import was barfing for me. In general, I had weirdness with pre-existing imports/scoping here for reasons I'm not clear on.